andrea/js/app.js

```diff
@@ -0,0 +1,96 @@
+// buttons //
+let signUpButton = document.querySelector("#navbar-sign");
```

Comment:
These don't need to be `let` but `const` instead. Remember `let` is only for variables that will be mutated.